app/main.py

```diff
@@ -1 +1,36 @@
-# write your code here
+from __future__ import annotations
+
+
+class Animal:
+    alive = []
+
+    def __init__(self,
+                 name: str,
+                 health: int = 100,
+                 hidden: bool = False) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+        Animal.alive.append(self)
+
+    def __repr__(self) -> str:
+        return f"{{Name: {self.name}, " \
+               f"Health: {self.health}, " \
+               f"Hidden: {self.hidden}}}"
+
+    @staticmethod
+    def still_alive(animal: Animal) -> None:
+        if animal.health <= 0:
+            animal.alive.pop(animal.alive.index(animal))
```

Comment:
You can remove specified animal from the Animal class `alive` variable using `remove` method (same as in your __init__ method,  just use `remove`)